src/tests/ftest/server/daos_server_restart.py

```diff
@@ -0,0 +1,188 @@
+        (2)Restart server without pool been created, and verify.
+        (3)Create pools, containers and verify the pools.
+        (4)Perform an orderly DAOS shutdown, and Use cmd line tools to
+           re-format both SSDs and NVDIMMs.
+        (5)Verify after DAOS server restarted, it should appear as an empty
+           fresh installation.
+
+        :avocado: tags=all,pr,hw,large,server_test,server_restart
+        """
+
+        self.log.info("(1)Verify daos server pool list after started.")
+        self.verify_pool_list()
+        self.log.info("(2)Restart server without pool created and verify.")
+        self.restart_daos_server()
+        self.verify_pool_list()
+        self.log.info("(3)Create pools, containers.")
+        self.create_pool_and_container()
+        self.log.info("(4)Shutdown, restart and reformat the server")
+        self.restart_daos_server()
+        self.log.info("(5)Verify after server restarted.")
+        self.verify_pool_list()
+
+    def test_daos_io_server_restart(self):
+        """
+        JIRA ID: DAOS-3593.
+
+        Test Description: Test cmd to perform daos io_server restart.
+        Steps:
+        (1)Use the cmd line to perform a controlled shutdown from a
+           quiescent state (i.e. clients disconnected).
+        (2)Use the cmd line to perform a controlled shutdown from a
+           partially quiescent state (i.e. clients attached but no
+           transactions in progress).
+        (3)Force shutdown and restart the daos io-server.
+        (4)Verify pool list after forced shutdown and restart the
+           daos io-server.
+        (5)Use the cmd line to perform a controlled shutdown when the
+           daos cluster is incomplete (i.e. 1 of the 2 servers is down).
+
+        :avocado: tags=all,pr,hw,large,server_test,server_reformat
+        """
+
+        self.log.info(
+            "(1)Shutdown and restart the daos io-server "
+            "from a quiescent state.")
+        self.agent_managers[0].stop()
+        self.verify_pool_list()
+        self.restart_daos_io_server()
+        self.agent_managers[0].start()
+        self.log.info(
+            "(2)Shutdown and restart the daos io-server with pools "
+            "and containers created.")
+        self.create_pool_and_container()
+        pool_list = self.get_pool_list()
+        self.restart_daos_io_server()
+        self.log.info(
+            "(3)Force shutdown and restart the daos io-server.")
+        self.restart_daos_io_server(force=True)
+        self.log.info(
+            "(4)Verify pool list after forced shutdown and restart "
+            "the daos io-server.")
+        self.verify_pool_list(pool_list)
+
+        print(" ")
+        print("self.hostlist_servers= ", self.hostlist_servers)
+        print(" ")
+        hosts = self.hostlist_servers
+        self.hostlist_servers = hosts[-1]
+        print(" ")
+        print("(5)Restart daos io server for the last server on the cluster ")
```

Comment:
Why do you use print instead of self.log.info?

Reply:
same as above.